### DISCUSSION TITLE
OpenBSD & Chamilio 2.0

### DISCUSSION THREAD

**Discussion by kd4e** [chamilo/chamilo-lms#5668] [General] [answered]
Will these instructions to install Chamilo with OpenBSD remain valid when v2.0 is released?
[(https://www.ipv6.rs/tutorial/OpenBSD/Chamilo_LMS/)]
Thanks

**Comment by kd4e** [comment]
E-mail answer said Yes. Hooray! Thanks!

**Comment by ywarnier** [comment, marked as answer]
Hi @kd4e 
Not sure what e-mail you got, but these instructions will not work as is, no.
- PHP will be required in version 8.2 at the very least (probably 8.3)
- the DocumentRoot will have to point to the "public/" directory inside the Chamilo directory
- there might be more extensions necessary

v2 is not "just around the corner" yet, so there are still some moving requirements pending. We'll update the main installation manuals once we have a Release Candidate version (RC). Current instructions (in root README.md) are for development and testing only.
